save-cloud-common/src/nativeMain/kotlin/com/saveourtool/save/utils/FileUtils.kt

```diff
@@ -7,11 +7,46 @@ package com.saveourtool.save.utils
 import com.akuleshov7.ktoml.file.TomlFileReader
 import okio.FileSystem
 import okio.Path
+import okio.Path.Companion.toPath
+import platform.posix.*
 
+import kotlinx.cinterop.UnsafeNumber
 import kotlinx.serialization.serializer
 
 actual val fs: FileSystem = FileSystem.SYSTEM
 
+@OptIn(UnsafeNumber::class)
+actual fun Path.markAsExecutable() {
+    @Suppress("CAST_NEVER_SUCCEEDS") val mode: mode_t = (S_IRUSR or S_IWUSR or S_IXUSR or S_IRGRP or S_IROTH) as mode_t
```

Comment:
Are you sure?

This is `rwxr--r--`.

You probably should just read the existing attributes and just merge them with `ugo+x` (`a+x`).

If you can't read the existing attributes using Okio, just grab the default attribute set (0644) and apply the `x` bit (0644 -> 0755).

Reply:
Just moved existing code to make it Multiplatform. Decided to leave as is in order to not to break anything

Reply:
Ok then. Let's leave as-is.